target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8071-ax3600.dts

```diff
@@ -51,9 +51,7 @@
 	perst-gpios = <&tlmm 52 GPIO_ACTIVE_HIGH>;
 
 	pcie@0 {
-		wifi0: wifi@0,0 {
```

Comment:
@robimarko I'm a bit out of the look. Are these copy paste error? (i mean the extra wifi0 tag)